.github/actions/verify-badge/action.yaml

```diff
@@ -0,0 +1,257 @@
+name: Verify-Badge
+description: Perform verification to assist with UDS badge certification
+
+runs:
+  using: composite
+  steps:
+    - name: Test installing the package
+      shell: bash
+      run: |
+        set -uo pipefail
+
+        # Validate UDS Package/Config chart is valid
+
+        echo "ℹ️ Installing kubectl-validate..."
+        go install sigs.k8s.io/kubectl-validate@latest
+
+        echo "ℹ️ Validating uds-config chart..."
+        kubectl validate <(helm template chart chart/)
+        echo
+
+        # Setup
+
+        NAMESPACE=$(yq '.components[].charts[].namespace' common/zarf.yaml | uniq)
```

Comment:
same with this - a decent example is monorepoed stuff like https://github.com/defenseunicorns/uds-package-sigstore or UDS Core itself.

Reply:
Also postgres-operator, gitlab-runner and (soon) minio-operator config charts deploy things into multiple namespaces. For example there is a Package CR in the `gitlab-runner` and `gitlab-runner-sandbox` namespace. These are not referenced directly in the zarf component definition.

Reply:
Any suggestions on the best way to parse out all applicable namespaces?

Reply:
Not saying this is great or anything, but in my scenario, you could get the namespaces from the config chart helm templates. There are package CR's in any namespace that would need to be looked at in the examples we have today. So grabbing ns from Package resources out of the template. 

Then for @Racer159's case of mono repo like sigstore and uds-core, you could expand on that to check both `./chart` and `src/<package>/chart` where package is any folder under `src` 🤷